app.py

```diff
@@ -8,12 +8,18 @@
 from flask import jsonify, request, redirect
 from datetime import datetime as dt
 
-from settings import app
+from settings import *
 from parser.csvparser import Parser
 import database.mock_db as mock_db
 from database.dbmodel import Pool, db, Software, OperatingSystem, User, Reservation
 from statistics import statistics as stats
 
+from flask_mail import Message
```

Comment:
Błąd u mnie "unresolved reference flask_mail", nie aktualizowałaś środowiska czasem? A jak tak to dodaj nowe paczki do environment.yml